### PR TITLE
Enable blending on terrain pass 0

### PR DESCRIPTION
**Pull request by UltraHex**
Improves the quality of layered block textures with a very slight performance hit. Also fixes a minor graphical glitch or two.

Fixes GTNewHorizons/GT-New-Horizons-Modpack#15359.

### CODE REVIEW

**Comment by UltraHex** [comment]
Can be turned off in config for those who don't want the quality/performance tradeoff.

**Comment by UltraHex** [comment]
Not ready. We found a bug.

**Comment by Caedis** [comment]
@UltraHex is this the bug (check the top of leaves)?

**Comment by UltraHex** [comment]
Yes. That's the bug.

The only solution I could come up with is 8x-ing the renderlists for each chunk and switching based on the player's view angle. That's not an acceptable tradeoff.

Well, that or patching every single pass 0 block that has transparent pixels.

TL;DR: This change is too fundamental and can't be implemented correctly without extreme measures.

This PR is dead.

**Comment by Caedis** [comment]
@Dream-Master be sure to remove this from the dev branch

**Comment by UltraHex** [comment]
For those who wish to know:

What's happening is pass-0 blocks with transparency (like leaves) will have partially transparent pixels when mipmapped. These pixels will still update the depth buffer, occluding blocks from the same chunk behind them that get rendered after.

**Comment by Dream-Master** [comment]
yea i make a new pre tag tomorrow
